Core/include/Acts/Seeding/SeedFinder.ipp

```diff
@@ -201,7 +199,7 @@ void SeedFinder<external_spacepoint_t, grid_t, platform_t>::createSeedsForGroup(
 
     m_config.seedFilter->filterSeeds_1SpFixed(
         state.spacePointData, state.candidates_collector,
-        seedFilterState.numQualitySeeds, outIt);
+        seedFilterState.numQualitySeeds, std::forward<container_t>(outIt));
```

Comment:
ops ... just realized this is may be a `std::move` performed multiple times on the same object